src/views/Product.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>{{ $t("Product") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <main>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>{{ $t("Sync") }}</ion-card-title>
+          </ion-card-header>
+          <ion-item>
+            <ion-label>{{ $t("Sync products") }}</ion-label>
+            <ion-select :interface-options="customPopoverOptions" interface="popover">
+              <ion-select-option>Hourly</ion-select-option>
+              <ion-select-option>Every 6 hours</ion-select-option>
+              <ion-select-option>Daily</ion-select-option>
+            </ion-select>
+          </ion-item>
+           <ion-item>
+            <ion-label>{{ $t("Sync collections") }}</ion-label>
+            <ion-select :interface-options="customPopoverOptions" interface="popover">
```

Comment:
For now lets use all popovers as components instead of inline incase we can find a way to make them reusable.

Reply:
Ok sir